README.md

```diff
@@ -100,6 +99,17 @@ hive.allow-drop-table=true
 hive.allow-rename-table=true
 ```
 
+#### Incremental models
+
+The incremental strategy currently supported by this adapter is to append new records
+without updating/overwriting any existing data from the target model.
```

Comment:
Worth to document that implemented incremental strategy is `append`. I assume that is a default.

Reply:
There's no other strategy for `incremental` than to append / insert to the existing content.